keyboards/handwired/108keyTrackpoint/108keyTrackpoint.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "quantum.h"
+
+#define KEYMAP( \
```

Comment:
`KEYMAP` here should be `LAYOUT`.

QMK has changed the terminology here since `handwired/trackpoint` was added.

```suggestion
#define LAYOUT( \
```